azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1656,7 +1663,14 @@ private void ajaxHandleUpload(final HttpServletRequest req, final HttpServletRes
     }
 
     if (this.lockdownUploadProjects) {
```

Comment:
It doesn't appear that this method has test coverage. 
I realize that there is not much REST API level automated tests currently. We will need to invest in this area.
For this change however, given the lack of test infrastructure, examples and potential regression risks, I will leave it to you to decide if you would like to add tests for it.